### PR TITLE
ref(devservices): dynamic rm prompt text

### PR DESCRIPTION
**Pull request by joshuarli**
Addresses https://github.com/getsentry/develop/pull/35#discussion_r428790609.

`devservices rm` takes nargs services, but the prompt text isn't generated accordingly.

This makes the experience similar to https://github.com/getsentry/sentry/pull/18514, read the PR description.

Also, this reminded me of this containers vs services wart that we assume a service has 1 container following naming schema: PROJECT_SERVICE. Notice how `selected_containers` is filtered from service names, and the resultant container names (w/o prefixes) are used in the prompt text as "services".

Ideally we should be able to take a service name and list all applicable containers, volumes, and networks.

So if anyone wants to block this by requesting a refactor for that, I could take a stab at it, but it might take a while.

### CODE REVIEW

**Review by markstory — COMMENTED** [review]
Looks good to me. Refactoring the duplication is something that can be done separately imo.

**Review by markstory — APPROVED** [review]

File: src/sentry/runner/commands/devservices.py [hunk 1]
```diff
@@ -305,27 +313,63 @@ def down(project, service):
...
-        if container.name.startswith(prefix):
-            if not service or container.name[len(prefix) :] in service:
-                click.secho("> Removing '%s' container" % container.name, err=True, fg="red")
-                container.stop()
-                container.remove()
+    for container_name in containers:
+        click.secho("> Removing '%s' container" % (prefix + container_name), err=True, fg="red")
+        container = client.containers.get(prefix + container_name)
+        container.stop()
+        container.remove()
```
Comment (tail) [inline src/sentry/runner/commands/devservices.py:372]: FYI, there are references to `service` still below.

Comment (joshuarli) [reply src/sentry/runner/commands/devservices.py:372]: Oh, good catch - thanks! Love this about python, haha.

Comment (joshuarli) [reply src/sentry/runner/commands/devservices.py:372]: https://github.com/getsentry/sentry/pull/18975

**Pull request merged**